### DISCUSSION TITLE
s2复制原始数据缺失

### DISCUSSION THREAD

**Discussion by wangdp96** [antvis/S2#2789] [General]
在不使用SheetComponent组件的情况下，如何手动配置复制数据的格式，eg:copyData(s2,',',false),配置为树状图，且折叠展示，复制后数据为折叠数据，被折叠部分未复制到

**Comment by lijinke666** [comment]
目前不支持复制未渲染的节点, 树状折叠后的节点不会渲染, 可以另起一个 s2 实例展开全部节点然后获取复制的数据
